native/NativeSampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/ui/MainActivity.java

```diff
@@ -132,6 +132,15 @@ protected void onCreate(Bundle savedInstanceState) {
 	}
 
 	@Override
+	protected void logoutCompleteActions() {
+		super.logoutCompleteActions();
```

Comment:
Yeah, I think that's a good idea. Should we put this in `SalesforceDroidGapActivity` as well then?